core/cog-shell.c

```diff
@@ -173,6 +173,8 @@ string_array_contains (const char *haystack[],
     return FALSE;
 }
 
+G_DEFINE_QUARK(cog-shell-error-quark, cog_shell_error)
```

Comment:
Small nit: a space is missing between the macro name and the `(` character.

Also: Ouch! We are also using an inconsistent style of the name passed as argument to the `G_DEFINE_QUARK` macro. This one I blame on myself, there's three different styles:

```
% grep -r G_DEFINE_QUARK core/ platform/
core/cog-directory-files-handler.c:G_DEFINE_QUARK (cog-directory-files-handler-data, cog_directory_files_handler);
core/cog-directory-files-handler.c:G_DEFINE_QUARK (CogDirectoryFilesHandlerError, cog_directory_files_handler_error)
platform/pwl.c:G_DEFINE_QUARK (com_igalia_Cog_PwlError, pwl_error)
```

But yours is already using what seems to be “the usual GLib style”, so let's keep it and try to harmonize the rest later on separately.